client/containers/workflow-list/component.vue

```diff
@@ -140,27 +144,18 @@ export default {
 
       return getStartTimeIsoString(range, startTime);
     },
-    state() {
-      const { statusName } = this;
-
-      if (!this.statusName || statusName == 'ALL') {
-        return 'all';
-      }
-
-      return statusName === 'OPEN' ? 'open' : 'closed';
-    },
```

Comment:
assuming thats just a refactoring to encapsulate the domain logic.

Reply:
Its just been moved to getState helper function